### ISSUE TITLE
Can't Find BlockQuote.svelte

### ISSUE THREAD

**Issue by oncomouse** [bmschmidt/pandoc-svelte-components#7]
I was trying out your [bare-bones-blog](https://github.com/bmschmidt/bare-bones-blog) repo and I receive the following error when trying to display blog posts on the development server:

~~~
5:18:39 PM [vite] error while updating dependencies:
Error: Build failed with 1 error:
node_modules/esbuild/lib/main.js:1426:27: ERROR: [plugin: vite-plugin-svelte:optimize-svelte] ENOENT: no such file or directory, open '/home/andrew/Projects/bare-bones-blog/node_modules/pandoc-svelte-components/BlockQuote.svelte'
    at failureErrorWithLog (/home/andrew/Projects/bare-bones-blog/node_modules/esbuild/lib/main.js:1604:15)
    at /home/andrew/Projects/bare-bones-blog/node_modules/esbuild/lib/main.js:1056:28
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
~~~

It looks like [`src/lib/Blockquote.svelte`](https://github.com/bmschmidt/pandoc-svelte-components/blob/main/src/lib/Blockquote.svelte) needs to be renamed to `BlockQuote.svelte` for things to work.

I'm using Linux, node 19.6.0, and npm 8.19.3, if any of that makes a difference.

**Comment by bmschmidt** [comment]
Thanks, report much appreciated. You're right: the reason it hasn't affected me is that MacOS filenames are not case sensitive. (To the point where now git isn't even seeing when I rename the file, frustratingly.) I've tried to do the change in it in github, and published to npm as 0.2.1. Hopefully it works now; I don't have a linux testing environment set up, alas, right now.

**Comment by oncomouse** [comment]
Yep, that fixed it!
